.nvmrc

```diff
@@ -0,0 +1 @@
+node
```

Comment:
I believe here should be version instead, like
`14.18.1`

Reply:
it should run the latest version of node